### DISCUSSION TITLE
Turquoise dots cover shaping points

### DISCUSSION THREAD

**Discussion by WolfgangZ42** [devemux86/cruiser#171] [Q&A] [answered]
Importing a route into Cruiser we get:
<img width="400" src="https://github.com/devemux86/cruiser/assets/121806238/6bae5662-5845-4f5c-85b3-9aba3588cf7b"/>   

**What are the turquoise dots for? Did I miss something?**

I can help myself by leaving Cruiser and starting it again. Then I see:
<img width="400" src="https://github.com/devemux86/cruiser/assets/121806238/8fb6669c-d524-4652-9cde-bc76bee33067"/>   

So - really a minor problem!

**Comment by devemux86** [comment]
@WolfgangZ42 thanks for the report.

Probably you imported a Kurviger route with the "Overlay" option as well?

The turquoise dots are the waypoints.
We probably don't need them, except for GPX files?

**Reply by 0709wiwiwi** [reply]
Gpx ?  Not really. 
Or the turquoise "waypoint" dots in the **background** ? Probably a little hard to realise ?
As the (small) overlay (red) track optimally must still be shown **on top** of the (tick) (blue) route.
(= usually= by default set)

**Comment by devemux86** [comment]
I see from your other report (https://github.com/devemux86/cruiser/discussions/170) that it is a Kurviger route.
They are the waypoints.

Probably we do not need them with the "Overlay" option?
We just need to overlay the track.

**Reply by WolfgangZ42** [reply]
For my opinion the track as overlay should be sufficient.

**Reply by 0709wiwiwi** [reply]
Agree. Applied for both .kurviger file and gpx **nav**track file.

A gpx route import by the not snapped rtept (routepoints) can be presented differently (turquoise dots)  of course.
Not snapped dot condition = turqoise. " The snapped result " is than shown as a "serialised" "normal" shaping_Via point.

**Reply by devemux86** [reply]
It will be available in the next Cruiser update.

**Comment by devemux86** [comment, marked as answer]
> We just need to overlay the track.

Available in [Cruiser 3.0.20](https://play.google.com/store/apps/details?id=gr.talent.cruiser).
